lib/IDE/Refactoring.cpp

```diff
@@ -2601,6 +2601,46 @@ bool RefactoringActionLocalizeString::performChange() {
   EditConsumer.insertAfter(SM, Target->getEndLoc(), ", comment: \"\")");
   return false;
 }
+    
+bool RefactoringActionDocCommentBoilerplate::isApplicable(ResolvedCursorInfo Tok, DiagnosticEngine &Diag) {
+#warning Implement here
+    if (Tok.Kind != CursorInfoKind::ValueRef)
+        return false;
+    
+    return Tok.ValueD->getKind() == DeclKind::Func;
+}
+    
+bool RefactoringActionDocCommentBoilerplate::performChange() {
+#warning Implement here
+    if(CursorInfo.ValueD->getKind() == DeclKind::Func) {
+        swift::SourceLoc startLocation = CursorInfo.ValueD->getStartLoc();
+        
+        if(isa<FuncDecl>(CursorInfo.ValueD)) {
+            SmallVector<StringRef, 5> outputLines;
+            EditConsumer.accept(SM, startLocation, "/**\n<#Function Summary#>\n");
```

Comment:
We generally prefer `///` instead of `/**` for documentation comments in Swift.

Reply:
That's what I saw in the code base but the sample on the page https://bugs.swift.org/browse/SR-7294 had the ```/**``` format...

Reply:
I’d say stick with `///`